app/controllers/pages_controller.rb

```diff
@@ -2,6 +2,9 @@ class PagesController < ApplicationController
   def about
   end
 
+  def android
+  end
+  
```

Comment:
Not sure why this is complaining. This is following the format of the rest of pages_controller.rb